cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,37 @@
+class IFramePage {
+
+    private baseUrl: string
+    private cssUrl: string
+    private selectorIFrame: string
+    private selectorIFrameTitle: string
+    private selectorIFrameCssButton: string
+
+    constructor() {
+        this.baseUrl = "https://www.w3schools.com/html/html_iframe.asp"
+        this.cssUrl = "https://www.w3schools.com/css/default.asp"
+        this.selectorIFrame = "iframe[title='W3Schools HTML Tutorial']"
+        this.selectorIFrameTitle = "#main h1"
+        this.selectorIFrameCssButton = "[title='CSS Tutorial']"
+    }
+
+    visit() {
+        cy.visit(this.baseUrl)
+    }
+
+    private getFrameTitle() {
+        return cy.iframe(this.selectorIFrame).find(this.selectorIFrameTitle)
+    }
+
+    checkTitleInIFrame(expectedTitle: string) {
+        this.getFrameTitle().should('have.text', expectedTitle)
+    }
```

Comment:
This method shouldn't exist
you need to do is to place this assertion in the iframe.cy.ts file like this:
      ` iFramePage.getFrameTitle().should('have.text', expectedTitle)`